.travis.yml

```diff
@@ -38,18 +38,17 @@ jobs:
 
     - name: "Build documentation"
       script: |
-        yarn run typedoc && \
-        echo -e 'include:\n  - "_*"' > doc/_config.yml
+        yarn run typedoc && cp ./docs/index.html ./dist && \
```

Comment:
would you mind moving that to a new line? Two shell commands in the same line doesn't look good :)

Reply:
Already addressed in latest revision.